approved-book-list.json

```diff
@@ -1,15 +1,25 @@
 {
+  "api_version": 2,
   "approved_books": [
     {
       "repository_name": "osbooks-writing-guide",
       "style": "english-composition",
-      "tutor_only": false,
-      "books": [
-        {
-          "uuid": "ee7ce46b-0972-4b2c-bc6e-8998c785cd57",
-          "slug": "writing-guide"
-        }
-      ]
+      "platforms": ["rex"],
+      "versions": [{
+        "repository_name": "osbooks-writing-guide",
```

Comment:
Why is this field duplicated here? It seems like an oversight in the example spec if this is in the parent object and in here.